src/views/MainView.vue

```diff
@@ -190,21 +190,6 @@ export default {
       });
     });
     if (!window.ipc) return;
-
-    window.ipc.on("DATA_CHANGE", () => {
-      this.fetchItems();
-    });
-    window.ipc.on("CONFIG_CHANGE", () => {
```

Comment:
Did you test this by changing the settings while on the main view and ensuring it updated? An easy test case would be light/dark mode

Reply:
Thanks, I tested going to light mode and back to Dark mode and both work.